other-docs/guides/upgrading/v13.md

```diff
@@ -0,0 +1,66 @@
+
+To upgrade to Altis v13, edit your `composer.json` and change the version constraint for `altis/altis` and any local environment modules to `^13.0.0`. Note that the new default and recommended version of PHP for Altis v13 is now 8.0. Make sure you have tested your custom code and any additional plugins for compatibility with [PHP version 8.0](#php-80).
+
+```json
+{
+	"require": {
+		"altis/altis": "^13.0.0"
+	},
+	"require-dev": {
+		"altis/local-server": "^13.0.0"
+	},
+	"config": {
+		"platform": {
+			"php": "8.0"
+		}
+	}
+}
+```
+
+Once you have made these changes run `composer update` and then run the `wp altis migrate` command:
+
+```sh
+# For cloud environments
+wp altis migrate
+
+# For local server
+composer server cli -- altis migrate
+```
+
+## Breaking Changes
+
+### PHP 8.1 ###
+
+Altis v13 fully supports PHP 8.1 in both local and cloud environments. Support for versions of PHP earlier than 8.0 has been removed in Altis v13. There are a number of [backward incompatible changes](https://www.php.net/manual/en/migration81.incompatible.php) in PHP 8.1 which should be addressed.
+
+Refer to our [PHP Version Guide](docs://guides/updating-php/) for up-to-date compatibility, testing and upgrading information.
+
+### PHP 7.4 ###
+
+The release of Altis v13 introduces the removal of support for PHP 7.4, follow the [PHP Version Guide](docs://guides/updating-php/) for information on how to upgrade to PHP 8.0 or PHP 8.1.
+
+
+## Headline Features
+
+### WordPress 6.0.1 ###
+
+Wordpress 6.0.1 brings with it a lot of performance improvements to the block editor, taxonomies and the cache API amongsts many others.
+
+The [WordPress 6.0 Field Guide](https://make.wordpress.org/core/2022/05/03/wordpress-6-0-field-guide/) is a great place to learn about what's new. Here are some of the highlights:
+
+- [Improved block locking](https://make.wordpress.org/core/2022/05/05/block-locking-settings-in-wordpress-6-0/)
+- [Enhanced template block customization](https://make.wordpress.org/core/2022/05/05/updates-to-the-wordpress-create-block-templating-system/)
+- [Full CRUD options for batch API cache operations](https://make.wordpress.org/core/2022/04/29/caching-improvements-in-wordpress-6-0/)
+- [Storage of file size in Media Metadata](https://make.wordpress.org/core/2022/05/02/media-storing-file-size-as-part-of-metadata/)
+- [Taxonomy performance improvements](https://make.wordpress.org/core/2022/04/28/taxonomy-performance-improvements-in-wordpress-6-0/)
+- [Support for the creation of multiple `theme.json` variations](https://make.wordpress.org/core/2022/05/03/global-styles-variations-in-wordpress-6-0/)
+
+
+## New User Documentation ###
+We have released a first version of user-oriented documentation for Altis.  These are less technical articles, geared more towards letting you know what all the buttons do. There are almost 30 articles to get you started and we will be improving the ones we have and adding more over time.
```

Comment:
We should have a link to these new docs here.

Reply:
@kovshenin I've added the link for the user-docs.